rbs/AssertionsRewriter.cc

```diff
@@ -197,6 +197,10 @@ optional<rbs::InlineComment> AssertionsRewriter::commentForNode(const unique_ptr
             } else if (regex_match(content.begin(), content.end(), untyped_pattern)) {
                 kind = InlineComment::Kind::UNSAFE;
             }
+        } else if (absl::StartsWith(content, "self as ")) {
```

Comment:
Is it expected that more space or tabs here would cause the annotation to be ignored?